app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -505,30 +507,26 @@ class Identification extends React.Component {
               </Form.Row>
               <Form.Row className="pb-3 ml-0">Languages that are not fully supported are indicated by a (*) in the below list.</Form.Row>
               <Form.Row>
-                <Form.Group as={Col} sm={24} md={12} id="primary_language_wrapper" className="pr-md-4 mb-2">
+                <Form.Group as={Col} sm={24} md={12} id="primary_language_wrapper" className="pr-md-1 mb-3">
                   <Form.Label htmlFor="primary-language-select" className="input-label">
                     PRIMARY LANGUAGE{schema?.fields?.primary_language?._exclusive?.required && ' *'}
                     <InfoTooltip tooltipTextKey="primaryLanguage" location="right"></InfoTooltip>
                   </Form.Label>
                   {this.renderLanguageSelect(true)}
+                  {this.renderPrimaryLanguageSupportMessage()}
                 </Form.Group>
-                <Form.Group as={Col} sm={24} md={12} id="secondary_language_wrapper" className="pl-md-4 mb-2">
+                <Form.Group as={Col} sm={24} md={12} id="secondary_language_wrapper" className="pl-md-1 mb-3">
                   <Form.Label htmlFor="secondary-language-select" className="input-label">
                     SECONDARY LANGUAGE{schema?.fields?.secondary_language?._exclusive?.required && ' *'}
                     <InfoTooltip tooltipTextKey="secondaryLanguage" location="right"></InfoTooltip>
                   </Form.Label>
                   {this.renderLanguageSelect(false)}
-                </Form.Group>
-              </Form.Row>
-              <Form.Row>
-                <Form.Group as={Col} sm={24} md={12} controlId="primary_language_support_message" className="pr-md-4 mb-0">
-                  {this.renderPrimaryLanguageSupportMessage()}
-                </Form.Group>
-                <Form.Group as={Col} sm={24} md={12} controlId="secondary_language_support_message" className="pl-md-4 mb-0">
                   {this.state.current.patient.secondary_language && (
-                    <i>
-                      <b>* Warning:</b> Not used to determine which language the system sends messages to the monitoree in.
-                    </i>
+                    <Alert variant="warning" className="mt-2 mb-0 px-3 py-2">
```

Comment:
can the Alert here and the one on line 248 be refactored into a method since they're pretty similar?

Reply:
yes, great idea! I added this for the contact ones too: https://github.com/SaraAlert/SaraAlert/pull/941/commits/e633d689177da71ff4bd503b90ba06bd9f74dffa